app/web/src/features/communities/EditCommunityInfoPage.tsx

```diff
@@ -45,21 +54,28 @@ interface UpdatePageData {
   communityId: string;
   content: string;
   pageId: string;
+  communityPhotoKey?: string;
 }
 
 export default function EditCommunityPage() {
   const classes = useStyles();
   const queryClient = useQueryClient();
-  const { control, handleSubmit, register } = useForm<UpdatePageData>();
+  const { control, handleSubmit, register, errors } = useForm<UpdatePageData>();
 
   const {
     error,
     isLoading,
     isSuccess,
     mutate: updatePage,
   } = useMutation<Page.AsObject, GrpcError, UpdatePageData>(
-    ({ content, pageId }) =>
-      service.pages.updatePage({ content, pageId: +pageId }),
+    (data) => {
+      const { communityPhotoKey, content, pageId } = data;
```

Comment:
```suggestion
    ({ communityPhotoKey, content, pageId }) => {
```
Could have destructured directly from the function parameter since `data` isn't being used/needed elsewhere?